demo/components/pagination/pagination-demo.html

```diff
@@ -10,11 +10,12 @@
                 (numPages)="smallnumPages = $event"></pagination>
     <pre class="card card-block card-header">The selected page no: {{currentPage}}/{{smallnumPages}}</pre>
     <button type="button" class="btn btn-info" (click)="setPage(3)">Set current page to: 3</button>
+    <button type="button" class="btn btn-info" (click)="increaseItems(3)">Increase items with: 3</button>
```

Comment:
this one is not obvious for end user
you need to press button to increase amount of pages
filling like pager is brokean